src/App.tsx

```diff
@@ -8,11 +9,35 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type Props = {
+};
```

Comment:
```suggestion
```

You can just write an empty object in the generic

Reply:
Agree